Modelica/Mechanics/Rotational.mo

```diff
@@ -3375,7 +3375,7 @@ in the User's Guide of the Rotational library.
         Modelica.Mechanics.Rotational.Interfaces.PartialElementaryTwoFlangesAndSupport2;
 
       parameter Real tau_pos[:, 2]=[0, 1]
-        "[w,tau] Positive sliding friction characteristic (w>=0)";
+        "Positive sliding friction characteristic [N.m] as function of w (col. 1, w>=0)";
```

Comment:
What does `col. 1` mean?

Reply:
Column 1.
Yes, I agree there is missing col. 2. Maybe not that good idea. I will delete it.

I have introduced the decription "as function of" in the hope it clearly states which variable is in the first column and which in the second one. But may be there exists some common description for such cases. 
Actually, it would be nice to have some standard parameter description for
- 1d table with single y column
- 1d table with multi y columns

Do we need this also for other table types?
IMO the solution should include information on
- which column the variable is,
- unit of variables
- and start with upper case :-)

Something like "Variable y (unit y)  _as function of_ / _versus_ / _over_ variable x (unit x).